<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteAssignments</class>
 <widget class="QDialog" name="DeleteAssignments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Assignments</string>
  </property>
  <widget class="QListWidget" name="lw_Assignments">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Delete">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SelectAll">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>370</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SelectNone">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>DeleteAssignments</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
